Description: Don't detect upstream release version from Git, fails with the veyon packaging Git.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: not needed, Debian-specific

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -67,55 +67,11 @@
 include(PchHelpers)
 include(SetDefaultTargetProperties)
 
-find_package(Git)
-
-if(GIT_FOUND)
-	execute_process(COMMAND "${GIT_EXECUTABLE}" describe --tags
-		WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-		OUTPUT_STRIP_TRAILING_WHITESPACE
-		OUTPUT_VARIABLE VERSION_STRING)
-	string(REGEX REPLACE "^v([0-9]+)\\..*" "\\1" VERSION_MAJOR "${VERSION_STRING}")
-	string(REGEX REPLACE "^v[0-9]+\\.([0-9]+).*" "\\1" VERSION_MINOR "${VERSION_STRING}")
-	string(REGEX REPLACE "^v[0-9]+\\.[0-9]+\\.([0-9]+).*" "\\1" VERSION_PATCH "${VERSION_STRING}")
-
-	# determine build number to use in NSIS installer and resource files
-	execute_process(COMMAND "${GIT_EXECUTABLE}" describe --tags
-		COMMAND cut -d "-" -f2
-		WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-		OUTPUT_STRIP_TRAILING_WHITESPACE
-		OUTPUT_VARIABLE VERSION_BUILD)
-	if(NOT VERSION_BUILD GREATER 0)
-		set(VERSION_BUILD 0)
-	endif()
-
-	# Get list of all committers from git history, ordered by number of commits.
-	# The CONTRIBUTORS file is used by AboutDialog. This information can be provided
-	# with -DCONTRIBUTORS=/path/to/CONTRIBUTORS instead. For instance, to generate
-	# this file for version 3.0.2, the command is:
-	#   git shortlog -sne v3.0.2 | cut -c8-
-	set(CONTRIBUTORS "${CMAKE_BINARY_DIR}/CONTRIBUTORS")
-	if(NOT EXISTS "${CONTRIBUTORS}")
-		execute_process(COMMAND "${GIT_EXECUTABLE}" shortlog -s d160d147165271516589c304cb1b8f5e48f8527d..HEAD
-			COMMAND cut -c8-
-			COMMAND sort -f
-			OUTPUT_FILE "${CONTRIBUTORS}"
-			WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
-			TIMEOUT 10)
-	endif()
-
-endif()
-
-# can't retrieve version information as not building from Git repository?
-if(NOT VERSION_STRING)
-	set(VERSION_MAJOR 4)
-	set(VERSION_MINOR 7)
-	set(VERSION_PATCH 5)
-	set(VERSION_BUILD 0)
-	set(VERSION_STRING "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
-else()
-	# remove leading character from tag name
-	string(REPLACE "v" "" VERSION_STRING "${VERSION_STRING}")
-endif()
+set(VERSION_MAJOR 4)
+set(VERSION_MINOR 7)
+set(VERSION_PATCH 5)
+set(VERSION_BUILD 0)
+set(VERSION_STRING "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
 
 # set up compiler version variable
 string(REGEX REPLACE "\\.[0-9]$" "" COMPILER_VERSION_MAJOR_MINOR ${CMAKE_CXX_COMPILER_VERSION})
